lib/wallaby/metadata.ex

```diff
@@ -1,9 +1,5 @@
 defmodule Wallaby.Metadata do
-  @moduledoc """
-  Metadata is used to encode information about the browser and test. This
-  information is then stored in a User Agent string. The information from the
-  test can then be extracted in the application.
-  """
```

Comment:
While we already have the comment, maybe we keep it as a code comment? I'm not a big fan of those, but a high level comment for the module might help for new contributors? Wdyt?